profiles/base-crafting.yaml

```diff
@@ -1819,4 +1879,9 @@ crafting_recipes:
   chapter: 6
   container: bowl
   herb1: belradi
-  herb1-stock:
\ No newline at end of file
+<<<<<<< HEAD
```

Comment:
Fixed.

-- 
Regards,
Dr. Walter Wesley "Wes" Snyder V
484-300-4833 (h)
484-431-3355 (m)
484-302-7006 (f)

On 09/10/2016 09:53 PM, Robert Herbig wrote:
>
> In profiles/base-crafting.yaml 
> <https://github.com/rpherbig/dr-scripts/pull/776#discussion_r78286689>:
>
> > \ No newline at end of file
> > +<<<<<<< HEAD
>
> More merge artifacts.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/rpherbig/dr-scripts/pull/776/files/ff4f0f743a88719df99147c8d373c887fe41441e#r78286689>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AU8XXIOaiEEB5UBd_JBn2bJVY1NXeZ3mks5qo18vgaJpZM4J531I>.
>